packages/confirmation-modal/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -59,14 +61,24 @@ export const ConfirmationModal = React.forwardRef(
             // eslint-disable-next-line jsx-a11y/no-autofocus
             autoFocus
             darkMode={darkMode}
-            data-testid="lg-confirmation-modal-confirmation-input"
+            data-testid={LGIDS_CONFIRMATION_MODAL.input}
```

Comment:
should this use the `lgid`? I thought that was only for `data-lgid`? Or are we defaulting the same string for both?